internal/resolution/manifest/maven.go

```diff
@@ -139,15 +131,40 @@ func (m MavenManifestIO) Read(df lockfile.DepFile) (Manifest, error) {
 				Version:     string(project.Version),
 			},
 		},
-		Requirements: imports,
+		Requirements: requirements,
 		Groups:       groups,
 		EcosystemSpecific: MavenManifestSpecific{
-			Properties:      properties,
-			OriginalImports: originalImports,
+			Properties:                 properties,
+			RequirementsWithProperties: reqsWithProps,
 		},
 	}, nil
 }
 
+// ContainsProperty returns whether a string contains property placeholder.
+func ContainsProperty(str maven.String) bool {
```

Comment:
Add some examples of property placeholders in the docstring (or add a unit test for it.)